### ISSUE TITLE
Require cycle in master

### ISSUE THREAD

**Issue by kdo1234** [wonday/react-native-orientation-locker#161]
Master has this warning:

```
Require cycle: node_modules\react-native-orientation-locker\index.js -> node_modules\react-native-orientation-locker\src\OrientationLocker.js -> node_modules\react-native-orientation-locker\index.js

Require cycles are allowed, but can result in uninitialized values. Consider refactoring to remove the need for a cycle.
```

A fix seems to be changing line 2 of src/OrientationLocker.js from this:
`import Orientation from '..';`
to this:
`import Orientation from './orientation';`

**Comment by tapz** [comment]
Would it be time to fix this?

**Comment by wonday** [comment]
@tapz 
I have commit it to master, please test, 
if no problem, I will release it.

**Comment by tapz** [comment]
@wonday Seems to work.

**Comment by jonnyn** [comment]
let me know when this fix will be released

**Comment by wonday** [comment]
I have released it to v1.3.1.
